shader-examples/fresnel.frag

```diff
@@ -0,0 +1,81 @@
+#version 430
```

Comment:
No need to commit this file, keep it in your myShaders folder that is not pushed on the repo